### DISCUSSION TITLE
Linked vs separate account - Raspberry Pi

### DISCUSSION THREAD

**Discussion by phred2** [bbernhard/signal-cli-rest-api#516] [General]
Hi,

Want to send alarm message from a Raspberry Pi to my mobile phone that has a Signal account. Is it best to try and link the RPi to the existing account of my phone (QR code)  or try to creating a new Signal account for the RPi (I would use a landline number that can receive text messages).

 Thanks for any help.

Fred

**Comment by bbernhard** [comment]
Both should work. It depends what you are most comfortable with. I prefer a separate number, as the Signal Messenger API is then clearly separated from my (work) phone.

**Reply by patienttruth** [reply]
bbernhard, I have a second phone number that I've registered with this API. That works good. 

I'm setting up a second HA instance at another property and want to use the same number.

My question is: if I register using 

```curl -X POST -H "Content-Type: application/json" --data '{"use_voice": false}' 'http://192.168.88.100:8080/v1/register/+1xxxxx'```

will Signal boot my first HA copy of [signal-cli-rest-api](https://github.com/bbernhard/signal-cli-rest-api)? in this case should i be using the other [option](https://github.com/bbernhard/signal-cli-rest-api/blob/master/doc/HOMEASSISTANT.md#alternative-2-use-your-existing-mobile-phone-number)?

**Comment by MilamberValheru** [comment]
Hi Bernhard, 
if I use scr-api with a separate number. Is it possible to link a desktop-signal to that number (mainly for monitring purposes)?
